### DISCUSSION TITLE
Please HELP..

### DISCUSSION THREAD

**Discussion by Johnny112786** [phhusson/treble_experimentations#1741] [General]
Hi.. Does anyone know how to choose the correct GSI to my device from this list https://github.com/phhusson/treble_experimentations/wiki/Generic-System-Image-%28GSI%29-list ? I have Xiaomi Mi 10T 5G with Android 10 combined MIUI 12 [i guess].. I don't know much about this, could i anyway install an older Android [Just android, no MIUI] like Android 9 or 8 ? There's a lot on stuff they changed in An-10 i don't like..

Thanks ahead..

**Comment by phhusson** [comment]
No GSI can't be used to go back to an older android version.

Le mer. 3 mars 2021 à 23:07, Johnny112786 <notifications@github.com> a
écrit :

**Reply by Johnny112786** [reply]
> No GSI can't be used to go back to an older android version. Le mer. 3 mars 2021 à 23:07, Johnny112786 <notifications@github.com> a écrit :
> […](#)
> Hi.. Does anyone know how to choose the correct GSI to my device from this list https://github.com/phhusson/treble_experimentations/wiki/Generic-System-Image-%28GSI%29-list ? I have Xiaomi Mi 10T 5G with Android 10 combined MIUI 12 [i guess].. I don't know much about this, could i anyway install an older Android [Just android, no MIUI] like Android 9 or 8 ? There's a lot on stuff they changed in An-10 i don't like.. Thanks ahead.. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#1741>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAA4OWBOJL5RPFDC245FS3TB2XJRANCNFSM4YSB2NFQ> .

Hi.. thanks for your reply.. 

So is there any way i can install Android 7,8, or 9 to my device? even stock ROM or so? [i really not know much about it, but to my older Galaxy S6 i did installed some new ROMS and know how to use the ADB and stuff..]

And if not at all, can i install the same Android version to it [10, but without the MIUI..]? 


Thanks a lot..
